### ISSUE TITLE
How do I enable fancy Index module?

### ISSUE THREAD

**Issue by munntjlx** [byjg/docker-nginx-extras#1]
Do I just modify the dockerfile and add --with-http_module_fancy_index?

**Comment by byjg** [comment]
I dont think so. I read the module documentation at https://github.com/aperezdc/ngx-fancyindex and the process seems to a bit more complex than you said:

Read here the step by step:
https://github.com/aperezdc/ngx-fancyindex#building
